src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,7 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="SynchronizingWithClassView" xml:space="preserve">
     <value>Synchronizing with {0}...</value>
   </data>
+  <data name="FullSolutionAnalysisOff" xml:space="preserve">
+    <value>VS turned off full solution analysis due to memory concern. please re-start VS if you want full solution analysis back.</value>
```

Comment:
"Full solution analysis *has been* disabled due to low memory. Restart *Visual Studio* to restore full solution analysis."?